common/p_setup.cpp

```diff
@@ -1462,8 +1464,52 @@ void P_LoadBlockMap (int lump)
 	blockmap = blockmaplump+4;
 }
 
-
-
+/*
+* @brief P_GenerateUniqueMapHash
+* 
+* Creates a unique map hash used to identify a unique map.
+* Based on a few key lumps that makes a map unique.
+* 
+* @param things - Things lump offset
+* @param linedefs - Linedefs lump offset
+* @param sidedefs - Sidedefs lump offset
+* @param vertexes - Vertexes lump offset
+* @param segs - Segs lump offset
+* @param ssectors - SSectors lump offset
+* @param sectors - Sectors lump offset
+*/
+void P_GenerateUniqueMapHash(int things, int linedefs, int sidedefs, int vertexes,
```

Comment:
You don't need this many parameters.  The order of lumps for a given map is fixed, all that really matters is if it has a behavior lump or not.  I would recommend passing in just the base lump id and then behavior true/false.

Reply:
Pared this down to a reasonable amount of params.